### PR TITLE
docs(readme): Add link to pictogram package

### PR DESCRIPTION
**Pull request by connor-leech**
Added a link to the pictogram package in the readme.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -52,6 +52,7 @@ we support!
 | [`@carbon/colors`](./packages/colors)         | Work with IBM Design Language colors                                                                                                                |
 | [`@carbon/grid`](./packages/grid)             | Build layouts using the new 16 column grid system                                                                                                   |
 | [`@carbon/icons`](./packages/icons)           | Iconography assets. We also offer support in: [React](./packages/icons-react), [Angular](./packages/icons-angular), and [Vue](./packages/icons-vue) |
+| [`@carbon/pictograms`](.packages/pictograms)  | Pictogram assets |
```
Comment (joshblack) [inline README.md]: Would we want to mirror the tagline from above for React?

Comment (connor-leech) [reply README.md]: That's a good call. Do we offer pictograms in Angular/Vue yet?

Comment (joshblack) [reply README.md]: @connor-leech not that I'm aware of!

Comment (connor-leech) [reply README.md]: 👍

**Review by joshblack — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -52,6 +52,7 @@ we support!
 | [`@carbon/colors`](./packages/colors)         | Work with IBM Design Language colors                                                                                                                |
 | [`@carbon/grid`](./packages/grid)             | Build layouts using the new 16 column grid system                                                                                                   |
 | [`@carbon/icons`](./packages/icons)           | Iconography assets. We also offer support in: [React](./packages/icons-react), [Angular](./packages/icons-angular), and [Vue](./packages/icons-vue) |
+| [`@carbon/pictograms`](.packages/pictograms)  | Pictogram assets. We also offer support in: [React](./packages/pictograms-react)                                                                    |
```
Comment (emyarod) [inline README.md]:
```suggestion
| [`@carbon/pictograms`](./packages/pictograms)  | Pictogram assets. We also offer support in: [React](./packages/pictograms-react)                                                                    |
```

**Review by emyarod — COMMENTED** [review]
path seems incorrect

File: README.md [hunk 3]
```diff
@@ -52,6 +52,7 @@ we support!
 | [`@carbon/colors`](./packages/colors)         | Work with IBM Design Language colors                                                                                                                |
 | [`@carbon/grid`](./packages/grid)             | Build layouts using the new 16 column grid system                                                                                                   |
 | [`@carbon/icons`](./packages/icons)           | Iconography assets. We also offer support in: [React](./packages/icons-react), [Angular](./packages/icons-angular), and [Vue](./packages/icons-vue) |
+| [`@carbon/pictograms`](./packages/pictograms)  | Pictogram assets. We also offer support in: [React](./packages/pictograms-react)                                                                    |
```
Comment (emyarod) [inline README.md]:
```suggestion
| [`@carbon/pictograms`](./packages/pictograms) | Pictogram assets. We also offer support in: [React](./packages/pictograms-react)                                                                   |
```

**Review by emyarod — APPROVED** [review]
looks like we need to run `prettier` to pass CI, otherwise looks good to me

**Review by aledavila — APPROVED** [review]

**Pull request merged**